src/CgiRequestHandler.cpp

```diff
@@ -10,39 +12,66 @@
 #include "CgiRequestHandler.hpp"
 #include "Config.hpp"
 #include "Header.hpp"
+#include "HeaderFields.hpp"
 #include "IRequestManager.hpp"
 #include "ReadWriteFD.hpp"
 
 CgiRequestHandler::CgiRequestHandler(IRequestManager &manager, const Header &header, const RouteConfig &route)
 	: manager_(manager),
 	  responseFinished_(false),
+	  storedHeader_(header),
+	  bufferingRequestBody_(false),
+	  headersParsed_(false),
+	  useChunkedEncoding_(false),
+	  contentLength_(0),
+	  bytesSent_(0),
 	  startTime_(std::chrono::steady_clock::now())
```

Comment:
Style/opinion, but at least for simple non-class variables, specifying default initializers in the class header (`bool bufferingRequestBody_ = false;`) is cleaner than ever-growing constructor initializer lists. Of course things that depend on constructor params still need to be handled in constructor.